### PR TITLE
Adds cf-core to bower dependency list

### LINKED ISSUE #42: Add cf-core to bower.json?
`cf-core` is not a listed dep in the `bower.json`. While it's not required due to being a sub-dep of all the other CF components, this throws you off a bit when converting older projects. Since it doesn't really hurt and adds some clarity, should we add it as a direct dep?

### PR DESCRIPTION
**Pull request by jimmynotjim**
Adds cf-core to bower dependency list
 
## Additions
 
None
 
## Removals
 
None
 
## Changes
 
- manually added cf-core to list of user component options
- updated components message to clarify what steps the user is about to take
 
## Testing
 
- fetch and run npm link
- run yo cf and include cf-core when prompted for component options
- check `bower.json` for cf-core as listed dep
- check `/src/vendor` for cf-core for proper installation
 
## Review
 
- @Scotchester 
- @himedlooff 
- @contolini 
- @ascott1 
 
## Preview
 
screen shot 2015-03-24 at 5 10 53 pm
 
## Notes
 
- Including cf-core in the project deps adds clarity to the project
- Fixes #42

### CODE REVIEW

**Comment by Scotchester** [comment]
Oh, that makes it more clear why core was left out: Because it's required for CF to be useful at all, we didn't want it to be deselectable in the startup script.

**Comment by jimmynotjim** [comment]
Even if you de-select it at the prompt, it'll still be installed, which is why I find it kind of odd being there. Maybe we just tack it on to the list after the prompt has finished?

**Comment by anselmbradford** [comment]
Could you just change the wording on the prompt to something like:

`You're about to install CF Core. Which additional CF components would you like in your app?`

**Comment by jimmynotjim** [comment]
@anselmbradford I kind of like that combined with my idea of tacking it on. I'd still like to see it in the generated bower.json, but that message makes it clear that core will be installed by default no matter what.

**Comment by jimmynotjim** [comment]
I took @anselmbradford’s suggestion and updated the components message and manually added cf-core to the user’s provided options. This solves the issue of possibly de-selecting cf-core while also providing clarity on what steps the user is about to take.

File: app/lib/ignore.json [hunk 1]
```diff
@@ -4,6 +4,6 @@
   "cf-tabs",
   "cf-tables",
   "cf-demo",
-  "cf-colors",
-  "cf-core"
+  "cf-core",
+  "cf-colors"
```
Comment (Scotchester) [inline app/lib/ignore.json]: I don't think the order matters (looks like there was no particular order before), but we could avoid this diff if you put it back exactly the way it was.

Comment (jimmynotjim) [inline app/lib/ignore.json]: Whoops, thought I had done that. Good catch, just fixed and squashed down.

**Comment by jimmynotjim** [comment]
I'd like to squash out that commit changing the ignore file. Let me know if we're good to go before merging this and I'll do that.

**Comment by Scotchester** [comment]
:+1:

**Comment by jimmynotjim** [comment]
Ok, ready to merge from my side once it passes Travis CI again.

**Pull request merged**